make/00_mod.mk

```diff
@@ -33,14 +34,25 @@ oci_manager_image_name_development := cert-manager.local/trust-manager
 
 go_package_debian_main_dir := .
 go_package_debian_mod_dir := ./trust-packages/debian
-go_package_debian_ldflags := 
+go_package_debian_ldflags :=
 oci_package_debian_base_image_flavor := static
 oci_package_debian_image_name := quay.io/jetstack/cert-manager-package-debian
 oci_package_debian_image_tag := $(DEBIAN_BUNDLE_VERSION)
 oci_package_debian_image_name_development := cert-manager.local/cert-manager-package-debian
 debian_package_layer := $(bin_dir)/scratch/debian-trust-package
 oci_package_debian_additional_layers += $(debian_package_layer)
 
+go_package_debian_bookworm_main_dir := .
+go_package_debian_bookworm_mod_dir := ./trust-packages/debian
+go_package_debian_bookworm_ldflags :=
+oci_package_debian_bookworm_base_image_flavor := static
+oci_package_debian_bookworm_image_name := quay.io/jetstack/cert-manager-package-debian-bookworm
```

Comment:
@erikgb what do you think about this package name? The current one is `quay.io/jetstack/cert-manager-package-debian`.

I worry when we migrate to a cert-manager branded repo it'll be repetitive:

`quay.io/cert-manager/cert-manager-package-debian-bookworm`

Would `quay.io/jetstack/trust-package-debian-bookworm` work better?

Reply:
`quay.io/jetstack/trust-pkg-debian-bookworm`, to be shorter? 🤔

Reply:
Yes, I think this is a good rename. Also taken trust-manager maturing into consideration.